avalon/harmony/lib.py

```diff
@@ -55,7 +56,34 @@ def launch(application_path):
     self.application_path = application_path
 
     # Launch Harmony.
-    os.environ["TOONBOOM_GLOBAL_SCRIPT_LOCATION"] = os.path.dirname(__file__)
+    if os.getenv("TOONBOOM_GLOBAL_SCRIPT_LOCATION"):
+
+        avalon_harmony_startup_path = os.path.join(
+            os.getenv("TOONBOOM_GLOBAL_SCRIPT_LOCATION"), "TB_sceneOpened.js"
+        )
+
+        env_harmony_startup_path = os.path.join(
+            os.path.dirname(__file__), "TB_sceneOpened.js"
+        )
+
+        if not filecmp.cmp(avalon_harmony_startup_path,
+                           env_harmony_startup_path):
+            try:
+                shutil.copy(avalon_harmony_startup_path,
+                            env_harmony_startup_path)
+            except Exception as e:
+                self.log.error(e)
+                self.log.warning(
+                    "Failed to copy {0} to {1}! "
+                    "Defaulting to Avalon TOONBOOM_GLOBAL_SCRIPT_LOCATION."
+                        .format(avalon_harmony_startup_path,
+                                env_harmony_startup_path))
+
+                os.environ["TOONBOOM_GLOBAL_SCRIPT_LOCATION"] = \
+                    os.path.dirname(__file__)
```

Comment:
Sorry, I'm not a fan of `\` to escape line length. Better to assign to a variable.
```
path = os.path.dirname(__file__)
os.environ["TOONBOOM_GLOBAL_SCRIPT_LOCATION"] = path
```